components/ide/code/leeway.Dockerfile

```diff
@@ -5,6 +5,7 @@
 # we use latest major version of Node.js distributed VS Code. (see about dialog in your local VS Code)
 # ideallay we should use exact version, but it has criticla bugs in regards to grpc over http2 streams
 FROM node:12.21.0 AS node_installer
+# FROM node:12.22.1-alpine AS node_installer
```

Comment:
<details>
<summary>:warning: <strong>Commented-out Code is Committed.</strong> (click for details)</summary>

This pull request contains commented code which is not recommended when a version control repository (e.g., GitHub) is used. It is recommended to delete all commented instructions before committing to VCS.

**[More details](https://confluence.devfactory.com/x/yAV7HQ)**

## The improvement suggestion in this case
Consider avoiding committing commented code.
</details>

[//]: # (<a id="07734fc2-8512-4459-bae8-f7242a5c0180" data-finding-id="494601672" href="none" codereview-uuid-reserved-tag data-business-id="b8f23c7a-1d14-4bbb-9674-9b6f133e7a06" />)